bundles/framework/divmanazer/component/visualization-form/LineForm.js

```diff
@@ -57,16 +57,7 @@ Oskari.clazz.define(
             '<div class="column2">' +
             '<div class="column21">' +
             '<label>' + this.loc.color.label + '</label>' +
-            '<div class="color-grid">' +
-            '<div class="color-rectangle"></div>' +
-            '</div>' +
-            '<div class="color-label">' +
-            '<label>' + this.loc.color.labelOr + '</label>' +
-            '</div>' +
-            '<div class="color-source-selector">' +
-            '<label id="color-line-custom-rgb-label" for="color-line-custom-rgb">' + this.loc.color.labelCustom + '</label>' +
-            '</div>' +
-            '<div class="custom-colors"></div>' +
+            '<div id="color-picker-wrapper"></div>' +
```

Comment:
Replace id with class since there can be many color-pickers in DOM at the same time.